src/protocols/bdx/BdxTransferSession.h

```diff
@@ -365,14 +411,10 @@ class DLL_EXPORT TransferSession
     uint64_t mTransferLength       = 0; ///< 0 represents indefinite length
     uint16_t mTransferMaxBlockSize = 0;
 
-    // Used to store event data before it is emitted via PollOutput()
-    System::PacketBufferHandle mPendingMsgHandle;
-    StatusReportData mStatusReportData;
-    TransferInitData mTransferRequestData;
-    TransferAcceptData mTransferAcceptData;
-    BlockData mBlockEventData;
-    MessageTypeData mMsgTypeData;
-    TransferSkipData mBytesToSkip;
+    const uint8_t * mFileDesignator       = nullptr;
```

Comment:
This is non-owning. It points to memory possibly released. Why is this a non-owning pointer?